coral.py

```diff
@@ -485,7 +488,7 @@ def update(self):
 
             # Tell the bad news
 
-            display_highscore(len(snake.tail))
+            display_highscore(len(snake.tail) + snake.oranges)
```

Comment:
what about creating a function that returns the score `len(snake.tail) + snake.oranges` ?

```python
def score(snake):
    return len(snake.tail) + snake.oranges
```

Reply:
seems to be a better solution. implemented